TextToSpeech/impl/TTSManager.cpp

```diff
@@ -270,7 +270,7 @@ TTS_Error TTSManager::resume(uint32_t id) {
 TTS_Error TTSManager::shut(uint32_t id) {
     TTSLOG_TRACE("Shut");
 
-    if(m_speaker && m_speaker->cancelCurrentSpeech(id))
+    if((id != 0) && m_speaker && m_speaker->cancelCurrentSpeech(id))
```

Comment:
I think the call now should be cancelSpeech(id).

Reply:
Yes. I corrected the api now.